blockauth/apple/tests/test_notification_service.py

```
@@ -360,6 +452,190 @@ def run(self, payload):
+        ),
+        patch(
+            "blockauth.apple.notification_service.apple_setting",
+            side_effect=lambda key, default=None: {
+                "APPLE_SERVICES_ID": "com.example.services",
+                "APPLE_NOTIFICATION_TRIGGER": "_test._CaptureTrigger",
+                "APPLE_NOTIFICATION_MAX_AGE_SECONDS": 300,
+            }.get(key, default),
+        ),
+        patch("blockauth.utils.jwt.jwks_cache.requests.get", return_value=jwks_response),
+    ):
+        with pytest.raises(AppleNotificationVerificationFailed):
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)